src/main/java/core/stack/MateStack.java

```diff
@@ -1,19 +1,52 @@
 package core.stack;
 
+import java.util.EmptyStackException;
+
 public class MateStack<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final int CAPACITY_MULTIPLY = 2;
```

Comment:
I am not sure if it necessarily, but increase might be 1.5 as in ArrayList

Reply:
Vector documentation states that the capacity is doubled each time it needs to grow

Reply:
ok, thanks, I'll know)